ansible/roles/time/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Set timezone
+  timezone: name="{{ timezone }}"
+  notify: restart ntpd
+  tags:
+    - setup
+
+- name: Stop NTP service before running ntpdate
+  service: name=ntpd state=stopped
+  tags:
+    - setup
+
+- name: Run ntpdate
+  command: "ntpdate -t 5 {{ item }}"
+  notify: restart ntpd
+  with_items:
+    - "{{ ntp_servers }}"
```

Comment:
Just a curiosity, you are passing a nested list here. `with_items: "{{ntp_servers}}"` would be enough.
From https://github.com/ansible/ansible/issues/5913, ansible flattens the lists in `with_items`, so the result is the same.
As we already have both formats in our playbook, any one is ok.